doc/manual/customizing.rst

```diff
@@ -101,67 +294,12 @@ can be used to define a different viewer layout, add or remove plugins,
 add menu entries, add custom image or star catalogs, etc.  We will refer
 back to these functions in the sections below.
 
-=======================
 Workspace configuration
-=======================
+-----------------------
 
-Ginga has a flexible table-driven layout scheme for dynamically creating
-workspaces and mapping the available plugins to workspaces.  By changing
-a couple of tables via ``ginga_config.pre_gui_config()`` you can change
-the way Ginga looks and presents its content.
-
-If you examine the module `ginga.rv.main` you will find a layout table
-called ``default_layout``::
-
-    default_layout = ['seq', {},
```

Comment:
So setting `default_layout` like this will no longer work?

Reply:
No, it should work as before, if you are modifying from Python code, and assign over that variable.  There is also an example at the end of the customization page showing a way to make a customized startup script where you set your own layout from Python without having to know the global variable names from the `main` module.

Reply:
From reading https://ginga--763.org.readthedocs.build/en/763/manual/customizing.html#customizing-the-layout , it is still unclear to me if I have `default_layout` defined as well as the JSON/Python, which one will take precedence.

Reply:
If there is no custom layout defined, it will use the default layout.  But note that every time you close the program it saves the layout which becomes a custom layout (unless, as the documentation notes: you have set `save_layout = False` in your `general.cfg` or started the program with `--norestore`). I'll try to clarify this in the docs.

Reply:
@pllim, I think I've clarified this in the `customizing.rst`; can you read it over and LMK what you think.